portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -92,12 +97,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   // transforms the comments objects and the number of comments into JSON  
   private String convertToJsonUsingGson(ArrayList<Comment> comments, int numComments) {
+      String userId = "";
+      if (userService.isUserLoggedIn()) {
+          userId = userService.getCurrentUser().getUserId();
+      } else {
+          userId = null;
+      }
```

Comment:
Can you set it to null up here or just do `String userId;` and then you wouldn't need the else, right?

Reply:
you're very correct. fixed